### PR TITLE
CWP DOMA: language fixes by Bo

### PR DESCRIPTION
**Pull request by jouvin**

### CODE REVIEW

File: CWP/papers/HSF-CWP-2017-04_doma/latex/hsf-cwp-dm.tex [hunk 1]
```diff
@@ -306,15 +306,15 @@ \subsubsection{Object and Cloud Storage
...
-scale to large numbers of CPU core where now these applications
+scale to large numbers of CPU cores whereas in current models application
 concurrency is limited by the requirement to sequentially write events
 to files. Currently, the efficiency of of reading and writing events
-from object stores is sensitive to the size of the event. The smaller
-the event the worse is the efficiency.. Some of this may be mitigated by
-creating event bundles (i.e. several events in one object). The
+from object stores is sensitive to the size of the event---
+the smaller the event, the worse the efficiency is. Some of these potential inefficiencies
+may be mitigated by creating event bundles (\emph{i.e.} several events in one object). The
```
Comment (graeme-a-stewart) [inline CWP/papers/HSF-CWP-2017-04_doma/latex/hsf-cwp-dm.tex]: Generally I would suggest that you do not italicise Latinate abbreviations (most modern English style guides suggest this). If you do want to adopt this style, however, you'd better do it consistently. There definitely are some "e.g."s you'd need to fix.

**Comment by graeme-a-stewart** [comment]
Hi @jouvin - this is still WIP, right?

**Comment by jouvin** [comment]
It is probably not the last update but I'd suggest to merge this one. This will make further change diff easier.

**Comment by graeme-a-stewart** [comment]
Done!

**Pull request merged**